fast_llm/tools/cli.py

```diff
@@ -32,7 +32,6 @@ def fast_llm(args=None):
         sys.exit(1)
     except Exception:  # noqa
         logger.critical(traceback.format_exc())
-        sys.exit(1)
```

Comment:
I know we need this line to be removed for the debugger to work. You could do this instead:

```python
    except Exception:  # noqa
        if sys.gettrace():
            raise
        logger.critical(traceback.format_exc())
        sys.exit(1)
```

Reply:
That would work, we do need that line outside a debugger. Same thing needed for `ValidationError` above.